apps/docs/pages/guides/auth/auth-helpers/nextjs.mdx

```diff
@@ -79,7 +79,7 @@ NEXT_PUBLIC_SUPABASE_ANON_KEY=your-supabase-anon-key
 
 When using the Supabase client on the server, you must perform extra steps to ensure the user's auth session remains active. Since the user's session is tracked in a cookie, we need to read this cookie and update it if necessary.
 
-In Next.js Server Components, you can read a cookie, but you can't write back to it. Middleware on the other hand, allow you to both read a write to cookies.
+In Next.js Server Components, you can read a cookie, but you can't write back to it. Middleware, on the other hand, allows you to both read and write to cookies.
```

Comment:
Thanks @gurdulicluka! Simplifying this a bit more and reverting back to `allow` (I believe we're using Middleware as plural here).
```suggestion
Next.js Server Components allow you to read a cookie but not write back to it. Middleware on the other hand allow you to both read and write to cookies.
```